File: Server/TruckInfoModule/Models/FullLogDto.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace TransportInfoModule.Models
{
    public class FullLogDto
    {
        public string Code { get; set; }
        public bool IsActive { get; set; }
        public string Description { get; set; }
        public DateTime ActiveDateTime { get; set; }
        public DateTime? InactiveDateTime { get; set; }
    }
}

File: Client/app/src/main/java/com/nks/interactive/multimediapanel/models/transportInfo/FullLogDto.kt
package com.nks.interactive.multimediapanel.models.transportInfo

import java.time.LocalDateTime

data class FullLogDto(
    val code: String = "",
    val isActive: Boolean = false,
    val description: String = "",
    val activeDateTime: LocalDateTime,
    val inactiveDateTime: LocalDateTime? = null
)